builtin/credential/ldap/path_config_rotate_root_test.go

```diff
@@ -0,0 +1,60 @@
+package ldap
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+// This test relies on an external ldap server with a suitable person object (cn=admin,dc=planetexpress,dc=com)
+// with bindpassword "admin". - see the backend_test for more details.
+// This test will not run unless VAULT_ACC is set to something
+func TestRotateRoot(t *testing.T) {
+	if os.Getenv(logicaltest.TestEnvVar) == "" {
+		t.Skip("skipping rotate root tests because VAULT_ACC is unset")
+	}
+	ctx := context.Background()
+
+	b, store := createBackendWithStorage(t)
+	defer b.Cleanup(ctx)
+
+	// set up auth config
+	req := &logical.Request{
+		Operation: logical.UpdateOperation,
+		Path:      "config",
+		Storage:   store,
+		Data: map[string]interface{}{
+			"url":      "ldap://localhost:389",
+			"binddn":   "cn=admin,dc=planetexpress,dc=com",
+			"bindpass": "admin",
+			"userdn":   "dc=planetexpress,dc=com",
+		},
+	}
+
+	_, err := b.HandleRequest(ctx, req)
+	if err != nil {
+		t.Fatalf("failed to initialize ldap auth config: %s", err)
+	}
```

Comment:
Nit but no error doesn't mean the request succeeded so checking the response may help someone in the future:
```
	resp, err := b.HandleRequest(ctx, req)
	if err != nil {
		t.Fatalf("failed to initialize ldap auth config: %s", err)
	}
	if resp != nil && resp.IsError() {
		t.Fatalf("failed to initialize ldap auth config: %s", resp.Data["error"])
	}
```